addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -54,11 +54,11 @@ const UrlPickerUserValueWidget = InputUserValueWidget.extend({
         await this._super(...arguments);
         const linkButton = document.createElement('we-button');
         const icon = document.createElement('i');
-        icon.classList.add('fa', 'fa-fw', 'fa-external-link')
-        linkButton.classList.add('o_we_redirect_to');
-        linkButton.title = _t("Redirect to URL in a new tab");
+        icon.classList.add('fa', 'fa-fw', 'fa-external-link');
+        linkButton.classList.add('o_we_redirect_to', 'o_we_link', 'ms-1');
+        linkButton.title = _t("Preview this URL in a new tab");
         linkButton.appendChild(icon);
-        this.containerEl.appendChild(linkButton);
+        this.containerEl.after(linkButton);
```

Comment:
Mmh, a bit strange, but probably better than having to refactor the whole CSS indeed. Especially keeping a future refactoring in OWL later in mind :+1: